route/build/src/dataTypes.f90

```diff
@@ -158,32 +148,36 @@ MODULE dataTypes
    integer(i4b)             , allocatable  :: qhru_ix(:)   ! Index of hrus associated with runoff simulation (="qhru")
  end type remap
 
- ! simulated runoff data
- type, public :: runoff
-   integer(i4b)                            :: nTime         ! number of time steps
-   integer(i4b)                            :: nSpace(1:2)   ! number of spatial dimension
-   real(dp)                                :: time          ! time variable at one time step
-   real(dp)                 , allocatable  :: sim(:)        ! flux simulation (HM_HRU) at one time step (size: nSpace(1))
-   real(dp)                 , allocatable  :: sim2D(:,:)    ! flux simulation (x,y) at one time step (size: /nSpace(1),nSpace(2)/)
-   integer(i4b)             , allocatable  :: hru_id(:)     ! id of HM_HRUs or RN_HRUs at which runoff is stored (size: nSpace(1))
-   integer(i4b)             , allocatable  :: hru_ix(:)     ! Index of RN_HRUs associated with river network (used only if HM_HRUs = RN_HRUs)
-   real(dp)                 , allocatable  :: basinRunoff(:)! remapped river network catchment runoff (size: number of nHRU)
-   real(dp)                 , allocatable  :: basinEvapo(:) ! remapped river network catchment runoff (size: number of nHRU)
-   real(dp)                 , allocatable  :: basinPrecip(:)! remapped river network catchment runoff (size: number of nHRU)
+ ! mapping time step between two time series e.g., simulation time step vs runoff time step for one simulation time step
+ type, public :: map_time
+   integer(i4b), allocatable :: iFile(:)
```

Comment:
Add some comments about what each of these variables are.

Reply:
Added descriptions on `map_time` data type and its components